### DISCUSSION TITLE
[Parameter study]

### DISCUSSION THREAD

**Discussion by coskrrb2002** [idaholab/moose#24037] [Q&A Modules: General] [answered]
Hi,

I would like to know if there is any function for parameter study.

For example, in the same input file, I only want to adjust a single parameter value and get the calculation results.

input.i -> parameter = 1 => results_out_1.out
           -> parameter = 2 => results_out_2.out 
           -> parameter = 3 => results_out_3.out 
...
           -> parameter = 100 => results_out_100.out 

Is there any function like this?

**Comment by GiudGiud** [comment, marked as answer]
Hello

There is, it s part of the Stochastic Tools module
https://mooseframework.inl.gov/docs/site/syntax/ParameterStudy/index.html
https://mooseframework.inl.gov/docs/site/source/actions/ParameterStudyAction.html

Guillaume

**Reply by coskrrb2002** [reply]
Oh, thanks for helping.

I will take a look!
